lib/beacon/router.ex

```diff
@@ -86,6 +86,7 @@ defmodule Beacon.Router do
         live_session session_name, session_opts do
           get "/beacon_assets/css-:md5", BeaconWeb.AssetsController, :css, as: :beacon_asset, assigns: %{site: opts[:site]}
           get "/beacon_assets/js:md5", BeaconWeb.AssetsController, :js, as: :beacon_asset, assigns: %{site: opts[:site]}
+          get "/beacon_assets/tailwind_config:md5", BeaconWeb.AssetsController, :tailwind_config, as: :beacon_asset, assigns: %{site: opts[:site]}
```

Comment:
It'd be ideal if we could import the config from Beacon as a file from an URL, and not the current method of getting the content and doing a hack for importing it.

Something like

```js
const configPromise = import(tailwindConfigUrl);
```